pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -797,7 +797,7 @@ bool GSQLBackend::getTSIGKey(const DNSName& name, DNSName* algorithm, string* co
     d_getTSIGKeyQuery_stmt->reset();
   }
   catch (SSqlException &e) {
-    throw PDNSException("GSQLBackend unable to retrieve named TSIG key: "+e.txtReason());
+    throw PDNSException("GSQLBackend unable to retrieve TSIG key with name '" + name.toString() + "': "+e.txtReason());
```

Comment:
Why is toString() used here (and in some other places) instead of toLogString() ?

Reply:
I believed we logged TSIG key names with a dot in other palces, but a look at the tcpreceiver tells me we're not